posthog/models/feedback/survey.py

```diff
@@ -62,3 +52,16 @@ class Meta:
     end_date: models.DateTimeField = models.DateTimeField(null=True)
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     archived: models.BooleanField = models.BooleanField(default=False)
+
+
+@mutable_receiver(post_save, sender=Survey)
+def update_surveys_opt_in(sender, instance, **kwargs):
+    active_surveys_count = Survey.objects.filter(
+        team_id=instance.team_id, start_date__isnull=False, end_date__isnull=True, archived=False
+    ).count()
+    if active_surveys_count > 0 and instance.team.surveys_opt_in is (False or None):
```

Comment:
I'm not entirely sure if (False or None) works LOL, I think the below is more correct?

```suggestion
    if active_surveys_count > 0 and (instance.team.surveys_opt_in is False or instance.team.surveys_opt_in is None):
```

Reply:
no js-like schenanigans necessary here, everything you expect to be falsy is indeed falsy 😂 

```suggestion
    if active_surveys_count > 0 and not instance.team.surveys_opt_in:
```

Reply:
Raquel made the excellent point of checking for non-API mode surveys only here :)